src/tuf/tuf.cc

```diff
@@ -0,0 +1,202 @@
+#include "aktualizr-lite/tuf/tuf.h"
+
+namespace aklite {
+namespace tuf {
+
+AkHttpsTufRepoSource::AkHttpsTufRepoSource(std::string name_in, boost::property_tree::ptree& pt) { init(name_in, pt); }
```

Comment:
I would move `*TufRepoSource` declaration and definition to dedicated h/cc files.

Reply:
Done